datalad_deprecated/resources/website/tests/test.html

```diff
@@ -261,6 +262,24 @@
           assert.deepEqual(clickHandler({name: 'childfile', type: 'file'}), {next: '/firstdir/prevdir/curdir/childfile', type: 'assign'}, 'get link to child file');  
       });
 
+      
+      QUnit.test("Context Menu Renderers", function(assert) {
```

Comment:
I guess forgotten here?

Reply:
Ach, sorry about that, should be fixed now.